### ISSUE TITLE
Make script encryption workflow clearer in the export settings

### ISSUE THREAD

**Issue by avencherus** [godotengine/godot#10803]
With Windows 10 for a 64 bit export.  When using the encrypted GDScript option, is anyone else ending up with an executable that crashes when it starts?

**Comment by akien-mga** [comment]
This is an issue tracker, not a forum. Please describe the bug you are experiencing with some more details (e.g. run the game from cmd.exe, see what it says). Following the issue template would have been good too.

**Comment by volzhs** [comment]
@avencherus did you compile templates yourself?
if you want to use **Encrypted gdscript**, you need to compile templates manually.
https://github.com/ScyDev/godot/wiki/Export-Encrypt-Scripts
I'm using encrypted script mode with 2.1.4 personally, and I don't have an issue for it.

@akien-mga I tried to find this information on official documents, but couldn't find it.

**Comment by avencherus** [comment]
My apologies.  There is not much to it.

Whenever I export using the script export mode as _Encrypted_, the resulting executable does not run.

![godot_v2 1 4-stable_win64_2017-08-31_09-36-07](https://user-images.githubusercontent.com/13004169/29909787-31e5814c-8e30-11e7-9331-0ea776fd382f.png)

**Comment by avencherus** [comment]
@volzhs Oh I see, yeah I was using the template from the site.  Thanks volzhs.

**Comment by timoschwarzer** [comment]
I think this should be noted in the docs and there should be a warning message in the UI where yiu can enter the key in Godot.

**Comment by volzhs** [comment]
it's better to have a link button to official document for creating templates and using encrypted script mode, when encrypted mode is selected.

**Comment by akien-mga** [comment]
> I think this should be noted in the docs and there should be a warning message in the UI where yiu can enter the key in Godot.

Agreed. Something like:
```
Note: For this option to work, you need to build custom export templates hardcoding your script encryption key for your target platforms. See <link to docs>the documentation</link> for details.
```
And yeah, somehow should write that docs page :)

**Comment by akien-mga** [comment]
We have now entered release freeze for Godot 3.0 and want to focus only on release critical issues for that milestone. Therefore, we're moving this issue to the 3.1 milestone, though a fix may be made available for a 3.0.x maintenance release after it has been tested in the master branch during 3.1 development. If you consider that this issue is critical enough to warrant blocking the 3.0 release until fixed, please comment so that we can assess it more in-depth.
